src/ceph_fuse.cc

```diff
@@ -39,7 +39,12 @@ using namespace std;
 
 void usage()
 {
-  cerr << "usage: ceph-fuse [-m mon-ip-addr:mon-port] <mount point>" << std::endl;
+  cout <<
+"usage: ceph-fuse [-m mon-ip-addr:mon-port] <mount point> [OPTIONS]\n"
+"  -o <fuse options> please refer to the fuse document\n"
```

Comment:
Can't we get fuse to output its help text by passing a crafted argv to fuse_parse_cmdline?